nl/mail/chrome/messenger/messengercompose/composeMsgs.properties

```diff
@@ -309,7 +309,7 @@ windowTitle=Opstellen: %1$S
 
 ## String used by the dialog that ask the user to enter a subject
 sendMsgTitle=Bericht verzenden
-subjectDlogMessage=You did not specify a subject for this message. If you would like to provide one, please type it now.
+subjectDlogMessage=U heeft geen onderwerp gespecificeerd voor dit bericht. Als u dit alsnog wilt doe, typ het dan alstublieft nu.
```

Comment:
CORRECTION NEEDED, it should be: U heeft geen onderwerp gespecificeerd voor dit bericht. Als u dit alsnog wilt doen, typ het dan alstublieft nu.

- Typo: ''doen'' vs. ''doe''